README.md

```diff
@@ -61,7 +61,19 @@ tree, but store the results in two different trees in the output file.
 
     ranger.run("DTT_out.root")
 ```
+The same is possible using only the method 'add_selection'.
 ## Example 4:
```

Comment:
Please feel free to delete the individual flatten_tree, copy_tree and bpv_selection methods